cmd/authmate/main.go

```diff
@@ -276,11 +284,20 @@ It will be ceil rounded to the nearest amount of epoch.`,
 				AwsCliCredentialsFile: awcCliCredFile,
 			}
 
-			if err = agent.IssueSecret(ctx, os.Stdout, issueSecretOptions); err != nil {
-				return cli.Exit(fmt.Sprintf("failed to issue secret: %s", err), 6)
-			}
+			errChan := make(chan error, 1)
+			go func() {
+				errChan <- agent.IssueSecret(ctx, os.Stdout, issueSecretOptions)
+			}()
```

Comment:
Can we just create new context with timeout `ctx = context.WithTimeout(ctx, timeoutFlag)`?